conjur/wrapper/http_wrapper.py

```diff
@@ -9,14 +9,21 @@
 import base64
 import logging
 import re
+import ssl
 from enum import Enum
+from typing import Union
 from urllib.parse import quote
-import requests
+import asyncio
+from aiohttp import BasicAuth, ClientError, ClientResponseError, ClientSSLError, ClientSession
+import async_timeout
 import urllib3
 
 from conjur.errors import CertificateHostnameMismatchException, HttpSslError, HttpError, \
     HttpStatusError
 from conjur.api.endpoints import ConjurEndpoint
+from conjur.wrapper.http_response import HttpResponse
+
+REQUEST_TIMEOUT_SECONDS = 10
```

Comment:
why 10?

Reply:
@mbenita-Cyberark said some requests might take a while and it's ok, so 10 should be good.